### PR TITLE
Prebidmanager analytics adapter: fix console error when utm is null and collect page info

### PR DESCRIPTION
**Pull request by Prebid-Manager**
## Type of change

- [x] Bugfix
- [ ] Feature
- [ ] New bidder adapter   
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Does this change affect user-facing APIs or examples documented on http://prebid.org?
- [x] Other

## Description of change

- fix console error when utm data in local storage is null
prebid.js:3 Prebid ERROR: Prebid Manager Analytics: Error TypeError: Cannot read property 'length' of null

- add collecting of page info (domain & referrer)

```

Be sure to test the integration with your adserver using the [Hello World](/integrationExamples/gpt/hello_world.html) sample page.

- contact email of the adapter’s maintainer
- [ ] official adapter submission

For any changes that affect user-facing APIs or example code documented on http://prebid.org, please provide:

- A link to a PR on the docs repo at https://github.com/prebid/prebid.github.io/

## Other information
<!-- References to related PR or issue #s, @mentions of the person or team responsible for reviewing changes, etc. -->

### CODE REVIEW

File: modules/prebidmanagerAnalyticsAdapter.js [hunk 1]
```diff
@@ -83,7 +83,7 @@ function collectUtmTagData() {
     if (newUtm === false) {
       utmTags.forEach(function (utmKey) {
         let itemValue = localStorage.getItem(`pm_${utmKey}`);
-        if (itemValue.length !== 0) {
+        if (itemValue !== null && typeof itemValue !== 'undefined' && itemValue.length !== 0) {
```
Comment (harpere) [inline modules/prebidmanagerAnalyticsAdapter.js:86]: very minor but simpler to just `if (itemValue && itemValue.length !== 0) {`

File: modules/prebidmanagerAnalyticsAdapter.js [hunk 2]
```diff
@@ -99,6 +99,16 @@ function collectUtmTagData() {
...
 
+function collectPageInfo() {
+  const pageInfo = {
+    domain: window.location.hostname,
+  }
+  if (document.referrer) {
+    pageInfo.referrerDomain = utils.parseUrl(document.referrer).hostname;
+  }
+  return pageInfo;
+}
```
Comment (harpere) [inline modules/prebidmanagerAnalyticsAdapter.js:110]: would be nice to add a unit test for this.

**Review by harpere — CHANGES_REQUESTED** [review]

File: modules/prebidmanagerAnalyticsAdapter.js (hunk 2 above)
Comment (Prebid-Manager) [reply modules/prebidmanagerAnalyticsAdapter.js:110]: added a test in test/spec/modules/prebidmanagerAnalyticsAdapter_spec.js

File: modules/prebidmanagerAnalyticsAdapter.js (hunk 1 above)
Comment (Prebid-Manager) [reply modules/prebidmanagerAnalyticsAdapter.js]: edited this line. thank you.

**Review by harpere — APPROVED** [review]

**Pull request merged**